Monika After Story/game/script-topics.rpy

```diff
@@ -14271,3 +14271,32 @@ label monika_hot_springs:
     m 5dkbfb "Ahhh,{w=0.2} {nw}"
     extend 5dkbfa "just the thought of it makes me feel all tingly, [player]~"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_dying_same_day",
+            category=['philosophy', 'romance'],
+            prompt="Dying the same day",
+            random=True,
+            aff_range=(mas_aff.ENAMORED, None),
+            conditional="renpy.seen_label('monika_back_ups')",
```

Comment:
Just FYI, adding `conditional` w/o `action` property does nothing at best. I guess you want to start your topic with `random` property set to `False` (aka its default value, so you could just not specify it), and add `action=EV_ACT_RANDOM`. This way when the condition is met, your event will be set to random.

Reply:
Will do.